tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -253,7 +256,23 @@ def get_vuln_files_exception_dict(template):
     # best effort mode which invokes the mitigation instructions (VERW in this case) without a
     # guarantee that they clear the CPU buffers. If the host has the microcode update applied
     # correctly, the mitigation works and it is safe to ignore the "Vulnerable" message.
-    if global_props.cpu_codename == "INTEL_SKYLAKE" or template == "T2S":
+    #
+    # Guest on Intel Skylake with C3 template
+    # ---------------------------------------
+    # If the processor does not enumerate IA32_ARCH_CAPABILITIES.{FBSDP_NO,PSDP_NO,SBDR_SSDP_NO},
+    # the kernel checks its list of affected/unaffected processors and determines whether the
```

Comment:
is this a single list for both affected and unaffected?

Reply:
there are two lists: `cpu_vuln_blacklist` and `cpu_vuln_whitelist`. Will change it to plural.

Reply:
fixed.